backend/src/test/java/io/debezium/configserver/util/Infrastructure.java

```diff
@@ -157,7 +160,7 @@ public static ConnectorConfiguration getMySqlConnectorConfiguration(int id, Stri
                 .with("database.server.name", "dbserver" + id)
                 .with("database.history.kafka.bootstrap.servers", KAFKA_HOSTNAME + ":9092")
                 .with("database.history.kafka.topic", "dbhistory.inventory")
-                .with("server.id", "debezium_" + (5555 + id - 1));
+                .with("database.server.id", Long.valueOf(5555 + id - 1));
```

Comment:
There were two problems here.  

* The property should have been `database.server.id`.  This was probably masked previously because the UI and the connector would automatically generate a value on Debezium 1.9.  We now force the UI and Debezium 2.0 to no longer auto-generate a value and so it must be supplied here.
* The value needs to be of type `Long`, not of type `String` as previously provided.  This problem was again masked by the fact we used to auto-generate the value and now we don't via the schema generator changes upstream in the 1.9 branch..